slices/x11-common.yaml

```diff
@@ -0,0 +1,30 @@
+package: x11-common
+
+essential:
+  - x11-common_copyright
+
+slices:
+  data:
+    essential:
+      - lsb-base_bins
+    contents:
+      /etc/X11/Xreset:
+      /etc/X11/Xreset.d/:
+      /etc/X11/Xresources/x11-common:
```

Comment:
if this is the only data-like file you need, you may want to rename the slice to be `resources` instead